docs/learn/vocabulary/flavourisation.mdx

```diff
@@ -61,7 +61,7 @@ Again, the number of possible combinations between languages is almost endless,
 
 <NorthAndSouth />
 
-In addition, in the Northern flavourisation infinitives end in `-ť` instead of `-ti`, the nominative/accusative singular of adjectives should be `-e` instead of `-o` (:notr[dobre detja] instead of Southern :notr[dobro děte]). The Northern flavourisation favours the relative pronoun :isv[ktory] whereas the Southern flavourisation favours :isv[koj]. Likewise, in the case of the prefix **`vy`-/`iz`-**, the former is North-oriented, the latter South-oriented. In the Southern flavourisation the preposition :isv[dlja] „for” should be substituted with _`za`_.
+In addition, in the Northern flavourisation infinitives end in `-ť` instead of `-ti`, the nominative/accusative singular of adjectives should be `-e` instead of `-o` (:notr[dobre detja] instead of Southern :notr[dobro děte]). The Northern flavourisation favours the relative pronoun :isv[ktory] whereas the Southern flavourisation favours :isv[koj]. Likewise, in the case of the prefix `vy-`-/`iz-`, the former is North-oriented, the latter South-oriented. In the Southern flavourisation the preposition :isv[dlja] „for” should be substituted with :isv[za].
```

Comment:
```suggestion
In addition, in the Northern flavourisation infinitives end in `-ť` instead of `-ti`, the nominative/accusative singular of adjectives should be `-e` instead of `-o` (:notr[dobre detja] instead of Southern :notr[dobro děte]). The Northern flavourisation favours the relative pronoun :isv[ktory] whereas the Southern flavourisation favours :isv[koj]. Likewise, in the case of the prefix `vy-`/`iz-`, the former is North-oriented, the latter South-oriented. In the Southern flavourisation the preposition :isv[dlja] „for” should be substituted with :isv[za].
```